packages/app/components/FathomAnalytics.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { useEffect, Suspense } from "react";
+
+import { load, trackPageview } from "fathom-client";
+import { usePathname, useSearchParams } from "next/navigation";
+
+const siteId = process.env.NEXT_PUBLIC_FATHOM_SITE_ID ?? "";
+
+/**
+ * This is an approach suggested by Next maintainers.
+ * @see https://github.com/derrickreimer/fathom-client/issues/44
+ */
+const TrackPageView = () => {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
+
+  useEffect(() => {
+    if (!siteId) throw new Error("Site ID not set, skipping Fathom analytics");
```

Comment:
Can we show a console error instead of throwing an error?

This is throwing an error in development, where we don't need a siteId.